sources/packages/backend/apps/api/src/services/application-exception/application-exception.service.ts

```diff
@@ -63,12 +65,19 @@ export class ApplicationExceptionService extends RecordDataModelService<Applicat
       .leftJoin("exception.assessedBy", "assessedBy")
       .where("exception.id = :exceptionId", { exceptionId });
 
+    if (studentId || applicationId) {
```

Comment:
What is the benefit of inner join with application when only studentId alone is passed?

Reply:
`exception.innerJoin("exception.application", "application")`  is only need if there is an application id or student id

Reply:
My confusion here is, if there is no applicationId passed for this method, but studentId is passed line 72(`exception.innerJoin("exception.application", "application")`) will be executed and will there be any benefit of having this inner join for this situation?

Reply:
if I got your qn, correctly, in order to set the student, it is joined with the application, actually, this logic was already there,  just refactored to add the application logic
![image](https://github.com/bcgov/SIMS/assets/77353155/63a9f064-61a5-4274-a45f-3d7a23b1ce20),